packages/web/src/pages/sign-on-page/SignOnPage.tsx

```diff
@@ -107,7 +111,15 @@ const DesktopSignOnRoot = (props: RootProps) => {
             backgroundPosition: 'center'
           }}
         >
-          {isExpanded ? null : <AudiusValues />}
+          <Switch>
+            <Route exact path={SIGN_UP_APP_CTA_PAGE}>
+              {/* @ts-ignore box type incorrect */}
+              <Box as='img' w='100%' src={imagePhone} />
```

Comment:
can we add something to the types like `({ as: string } & any) | OtherBoxProps`?

Reply:
ideally i thought styled was smart enough to see the "as" prop and add "as" component props to itself. something we should improve with harmony